### ISSUE TITLE
`adddependencies` command may result in invalid manifest.xml

### ISSUE THREAD

**Issue by ShawnTalbert** [oracle/netsuite-suitecloud-sdk#182]
### Your environment
**OS** :  Ubuntu 18.04 (WSL)
**Node.js version** : 12.16.2
**suitecloud-cli version** : 1.0.2
**Terminal/CMD tool** : Webstorm terminal (WSL)

### Describe the bug
running `project:adddependencies` results in the following element added to the `<objects>` dependency list in `manifest.xml`:
```xml
<object>scriptid=customrecord_ns_ibe_req_types.value_2_t1247005_146|scriptid=customrecord_ns_ibe_req_types.value_4_t1247005_146|scriptid=customrecord_ns_ibe_req_types.value_3_t1247005_146</object>
```

Subsequently running `project:validate` results in the following error:
```
Validation failed.

An error occurred during manifest validation.
Details: The object scriptid=customrecord_ns_ibe_req_types.value_2_t1247005_146|scriptid=customrecord_ns_ibe_req_types.value_4_t1247005_146|scriptid=customrecord_ns_ibe_req_types.valu
e_3_t1247005_146 in the dependencies list contains characters that are not allowed.
File: ~/manifest.xml
```

### Expected Behavior
The `project:adddependencies` command should never result in a `manifest.xml` that fails validation.

### Additional context
What do I do with this? I'm not sure what that <object> content means - what's the workaround? Can I just safely remove this as a dependency in manifest.xml?

**Comment by alii-gator** [comment]
@ShawnTalbert can you please update the cli to the latest 1.0.4 version. We fixed a similar issue to what you have described in the latest release available. 
So it would be interesting to see if you can reproduce it or not.

**Comment by ShawnTalbert** [comment]
OK great. it looks like 1.0.4 no longer adds those funky dependency objects to my manifest, but it still adds invalid features, as shown below:
```
An error occurred during manifest validation.
Details: The VATORTAX_OVERHAULING feature defined in the manifest does not exist.
Details: The PHONETICSORT feature defined in the manifest does not exist.
Details: The PERLINETAXNOTTAX_OVERH feature defined in the manifest does not exist.
Details: The CANADA feature defined in the manifest does not exist.
Details: The RECEIVABLESSALESORDERS feature defined in the manifest does not exist.
Details: The COMMITABLEORDERS feature defined in the manifest does not exist.
Details: The NOTADVANCEDJOBS feature defined in the manifest does not exist.
Details: The CANADANOTTAX_OVERHAULING feature defined in the manifest does not exist.
Details: The VATNOTTAX_OVERHAULING feature defined in the manifest does not exist.
Details: The DOCUMENTS feature defined in the manifest does not exist.
File: ~/manifest.xml
```
But I think that's being fixed by another issue?

**Comment by alii-gator** [comment]
@ShawnTalbert Yes this will be fixed with #124 
So seems like updating to the latest version fixed the originaly mentioned issue, i will close this issue.

BTW you can just remove these invalid feature from manifest and the project will be deployed without any issue. This is more like a work around for now.
